plugins/dbqueuedb/dbqueuedb.c

```diff
@@ -537,6 +538,8 @@ static int stat_callback(int consumern, size_t length,
         }
         stats[consumern].has_stuff = 1;
         stats[consumern].depth++;
+        if (stats[consumern].depth == gbl_nonodh_queue_scan_limit)
```

Comment:
Make it so `0` disables?

Reply:
It should already - if it's 0 it'll never match that condition.  I'll make it more explicit in the code.